src/domain/collaboration/collaboration/collaboration.service.ts

```diff
@@ -773,13 +786,9 @@ export class CollaborationService {
 
   /**
    * Move callouts that are not in valid groups or flowStates to the default group & first flowState
-   * @param defaultGroupName
-   * @param defaultFlowStateName
    * @param callouts
    */
   public moveCalloutsToCorrectGroupAndState(
```

Comment:
moveCalloutsToCorrectGroupAndState --> moveCalloutsToDefaultGroupAndState, based on the description.

The notation of 'correct' is too ambiguous.